src/libs/OptionsListUtils.ts

```diff
@@ -22,41 +30,90 @@ import * as TaskUtils from './TaskUtils';
 import * as TransactionUtils from './TransactionUtils';
 import * as UserUtils from './UserUtils';
 
+type Tag = {enabled: boolean; name: string; accountID: number | null};
+
+type Option = {text: string; keyForList: string; searchText: string; tooltipText: string; isDisabled: boolean};
+
+type PayeePersonalDetails = {text: string; alternateText: string; icons: OnyxCommon.Icon[]; descriptiveText: string; login: string; accountID: number};
+
+type CategorySection = {title: string; shouldShow: boolean; indexOffset: number; data: Option[]};
```

Comment:
Wrap object keys.
```suggestion
type Tag = {
    enabled: boolean;
    name: string;
    accountID: number | null;
};

type Option = {
    text: string;
    keyForList: string;
    searchText: string;
    tooltipText: string;
    isDisabled: boolean;
};

type PayeePersonalDetails = {
    text: string;
    alternateText: string;
    icons: OnyxCommon.Icon[];
    descriptiveText: string;
    login: string;
    accountID: number;
};

type CategorySection = {
    title: string;
    shouldShow: boolean;
    indexOffset: number;
    data: Option[];
};
```

Reply:
should't be this done by prettier?

Reply:
There is a case when Prettier does not force the one-line nor expanded variant. It is kind of up to developer. So, if you will move at least one key, Prettier moves all other.